### ISSUE TITLE
Back button is too small to be usable (18x18)

### ISSUE THREAD

**Issue by fvsch** [elk-zone/elk#1836]
The back button used in detail views was surprisingly hard to click. I ended up missing it half the time. Turns out it's small: 18x18 pixels.

<img width="744" alt="Screenshot of a toot view with an element overlay from Chrome DevTools showing that the back arrow button is a link element with a 18x18 size" src="https://user-images.githubusercontent.com/243601/221838543-2510cda7-5a2b-4df7-b279-acfc078b6ef8.png">

Recommendations for bigger target areas include:

[WCAG 2.1 Success Criterion 2.5.5 Target Size (AAA)](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html):

> The size of the target for pointer inputs is at least 44 by 44 CSS pixels except when: [series of possible exceptions]

[Material Design 2](https://m2.material.io/design/usability/accessibility.html#layout-and-typography):

> ### Touch targets
>
> Touch targets are the parts of the screen that respond to user input. They extend beyond the visual bounds of an element. For example, an icon may appear to be 24 x 24 dp, but the padding surrounding it comprises the full 48 x 48 dp touch target.
>
> For most platforms, consider making touch targets at least 48 x 48 dp. […]
>
> ### Pointer targets
>
> Pointer targets are similar to touch targets, but apply to the use of motion-tracking pointer devices such as a mouse or a stylus. Consider making pointer targets at least 44 x 44 dp.

[Apple Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/components/menus-and-actions/buttons#best-practices):

> Make buttons easy for people to choose. On a touchscreen, buttons need a hit target of at least 44x44 points to accommodate a fingertip. On all screens, it’s essential to include enough space around a button so that people can visually distinguish it from surrounding components and content, whether people use touch, a pointer, or a system that expands a button when it’s in focus.

If Elk tries to be a touch-compatible UI, it should probably try to achieve touch targets of at least 44x44 CSS pixels. Currently the main sidebar buttons are a bit shy of that at 42.5x37.5, which is not ideal but might still work.

But buttons that are below ~30px of size, and especially as small as 18x18, are a clear UX issue and lead to a lot of missed clicks & taps.

**Comment by SaintPeter** [comment]
I would like to see the button be the full width of the column. I like Twitter's "Thread" header to show you you're inside a thread.  This entire area (in orange) is clickable/tapable:
![image](https://user-images.githubusercontent.com/553494/222544672-325ee4ce-f86e-4826-8456-62fa50e9052f.png)

**Comment by patak-dev** [comment]
I think having a big padding here like twitter has is a good idea 
<img width="594" alt="image" src="https://user-images.githubusercontent.com/583075/222559543-159efa66-86f1-4908-a901-71f609bcefbd.png">
But I don't see twitter working with full width, it only goes back when you press on the padding (even if it shows a pointer, which is quite strange). I think it would be confusing if clicking on the right of this bar would get you back 🤔

@SaintPeter let us know if you would like to work on adding the padding to the back button. It would also be good to add the same to the back arrow for wide screens (that appears on the left sidebar).

**Comment by SaintPeter** [comment]
It's interesting, it appears to be different on desktop and mobile.  You are correct that in mobile it is just the arrow, but on Desktop it's full width (as illustrated).

Regardless, I think a full column width clickable/touchable button would make the most sense for both desktop and mobile. It should ideally go all the way to the top of the screen.

RE: Working on it
I'm afraid I'm fully subscribed right now with my day job, so I can't commit to anything.

**Comment by patak-dev** [comment]
No problem, thanks for your input on the issue 🙏
